pages/books/_id/index.vue

```diff
@@ -1,20 +1,50 @@
 <template>
   <section class="single-book">
-      <h1>{{book.title}}</h1>
+    <div class="title">
+      <h1 v-show="shown.title" @dblclick="edit('title')">{{book.title}}</h1>
+      <input v-show="!shown.title" type="text" v-model="book.title" @blur="reset" @keyup.enter="reset">
+    </div>
     <div>
-        <img :src="book.img" class="book-img" :alt="book.title">
+      <img :src="book.img" class="book-img" :alt="book.title">
+    </div>
+    <div class="author">
+      <p v-show="shown.author" @dblclick="edit('author')">Autor: {{book.author}}</p>
+      <input v-show="!shown.author" type="text" v-model="book.author" @blur="reset" @keyup.enter="reset">
+    </div>
+    <div class="description">
+      <p v-show="shown.description" @dblclick="edit('description')">Opis: {{book.description}}</p>
```

Comment:
The behaviour of editing the corresponding fields of the book should be somewhere else. The idea of editing book in the same view where it's shown is not appropriate I think. Those two things should be separate. 

Maybe @LaYatiX and you should analyze the possible solutions one more time and choose the best which suits your needs? What do you think about it @Geecek ?

_I do not like the idea of switching between p and input. Could you consider please using only one thing?_

Reply:
I'm not quite sure what you mean by saying that I should consider using either p or input.
How is user supposed to custom anything without input fields?

Reply:
As I said I do not like the idea of editing and viewing the same in the same view. But I understand that it was a case so please do not bother with the comment above.  As far as I know, you are able to just use the input and remove p tag.